src/emcpy/plots/create_plot.py

```diff
@@ -355,6 +355,24 @@ def add_colorbar(self, label=None, orientation='horizontal',
             raise TypeError('Data object has colorbar set to False.' +
                             'Set obj.colorbar=True')
 
+    def add_stats_dict(self, stats_dict={'Stats': 'None'}, fontsize=12):
```

Comment:
@CoryMartin-NOAA and @aerorahul had a chat on this and even @RussTreadon-NOAA agrees that this duplicate code needs to be pulled out.
We should create an issue immediately and create a priority PR for it.